ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/styles.tsx

```diff
@@ -0,0 +1,158 @@
+  rgba(${maskColours.white}, 0.74) 68%,
+  rgba(${maskColours.white}, 0.68) 70%,
+  rgba(${maskColours.white}, 0.62) 72%,
+  rgba(${maskColours.white}, 0.56) 74%,
+  rgba(${maskColours.white}, 0.5) 76%,
+  rgba(${maskColours.white}, 0.44) 78%,
+  rgba(${maskColours.white}, 0.38) 80%,
+  rgba(${maskColours.white}, 0.32) 82%,
+  rgba(${maskColours.white}, 0.26) 84%,
+  rgba(${maskColours.white}, 0.2) 86%,
+  rgba(${maskColours.white}, 0.16) 88%,
+  rgba(${maskColours.white}, 0.11) 90%,
+  rgba(${maskColours.white}, 0.07) 92%,
+  rgba(${maskColours.white}, 0.04) 94%,
+  rgba(${maskColours.white}, 0.02) 96%,
+  rgba(${maskColours.white}, 0) 98%,
+  rgba(${maskColours.white}, 0) 100%`;
+
+const extraWideMask = `
+  rgba(${maskColours.white}, 0) 0%,
+  rgba(${maskColours.white}, 0.01) 2%,
+  rgba(${maskColours.white}, 0.02) 3%,
+  rgba(${maskColours.white}, 0.05) 4%,
+  rgba(${maskColours.white}, 0.08) 5%,
+  rgba(${maskColours.white}, 0.12) 6%,
+  rgba(${maskColours.white}, 0.16) 7%,
+  rgba(${maskColours.white}, 0.21) 8%,
+  rgba(${maskColours.white}, 0.26) 9%,
+  rgba(${maskColours.white}, 0.32) 10%,
+  rgba(${maskColours.white}, 0.38) 11%,
+  rgba(${maskColours.white}, 0.44) 12%,
+  rgba(${maskColours.white}, 0.5) 13%,
+  rgba(${maskColours.white}, 0.56) 14%,
+  rgba(${maskColours.white}, 0.63) 15%,
+  rgba(${maskColours.white}, 0.69) 16%,
+  rgba(${maskColours.white}, 0.74) 17%,
+  rgba(${maskColours.white}, 0.8) 18%,
+  rgba(${maskColours.white}, 0.85) 19%,
+  rgba(${maskColours.white}, 0.89) 20%,
+  rgba(${maskColours.white}, 0.93) 21%,
+  rgba(${maskColours.white}, 0.96) 22%,
+  rgba(${maskColours.white}, 0.98) 23%,
+  rgba(${maskColours.black}) 24%,
+  rgba(${maskColours.black}) 25%,
+  rgba(${maskColours.black}) 50%,
+  rgba(${maskColours.black}) 54%,
+  rgba(${maskColours.white}, 0.98) 56%,
+  rgba(${maskColours.white}, 0.96) 58%,
+  rgba(${maskColours.white}, 0.93) 60%,
+  rgba(${maskColours.white}, 0.89) 62%,
+  rgba(${maskColours.white}, 0.84) 64%,
+  rgba(${maskColours.white}, 0.8) 66%,
+  rgba(${maskColours.white}, 0.74) 68%,
+  rgba(${maskColours.white}, 0.68) 70%,
+  rgba(${maskColours.white}, 0.62) 72%,
+  rgba(${maskColours.white}, 0.56) 74%,
+  rgba(${maskColours.white}, 0.5) 76%,
+  rgba(${maskColours.white}, 0.44) 78%,
+  rgba(${maskColours.white}, 0.38) 80%,
+  rgba(${maskColours.white}, 0.32) 82%,
+  rgba(${maskColours.white}, 0.26) 84%,
+  rgba(${maskColours.white}, 0.2) 86%,
+  rgba(${maskColours.white}, 0.16) 88%,
+  rgba(${maskColours.white}, 0.11) 90%,
+  rgba(${maskColours.white}, 0.07) 92%,
+  rgba(${maskColours.white}, 0.04) 94%,
+  rgba(${maskColours.white}, 0.02) 96%,
+  rgba(${maskColours.white}, 0) 98%,
+  rgba(${maskColours.white}, 0) 100%`;
+
+export default {
+  maskedImageLtr: ({ mq }: Theme) =>
+    css({
+      maskSize: '100% 100%',
+      maskImage: `linear-gradient(
+        180deg, ${mobileImageMask})`,
+      [mq.GROUP_4_MIN_WIDTH]: {
+        position: 'absolute',
+        right: '0', // right for LTR
```

Comment:
I think we can use [inset-inline](https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-start), which should be similar to the `margin-inline` to position based on the `dir` set in the DOM. May help reduce some CSS duplication and then just have the `ltr/rtl` specific classes for the `linear-gradient`?

Reply:
Ah good call - I have updated :)